src/Form/DeveloperSyncForm.php

```diff
@@ -85,17 +85,27 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#open' => TRUE,
     ];
 
-    $output = '<p>' . $this->t('Developer synchronization will:') . '<p>';
-    $output .= '<ol>';
-    $output .= '<li>' . $this->t('Create Drupal users for any Apigee Edge developers that are in this Drupal system') . '</li>';
-    $output .= '<li>' . $this->t('Create developers in Apigee Edge for all users in this Drupal system that are not already in Apigee Edge') . '</li>';
-    $output .= '</ol>';
-    $output .= '<p>' . $this->t('Note that any Drupal users that are created will have a random password generated and will need to reset their password to log in. The "Run Developer Sync" button will sync the developers, displaying a progress bar on the screen while running. The "Background Developer Sync" button will run the developer sync process in batches each time <a href=":cron_url">cron</a> runs and may take multiple cron runs to complete.', [':cron_url' => Url::fromRoute('system.cron_settings')->toString()]) . '</p>';
-
-    $form['sync']['description']['#markup'] = $output;
+    $form['sync']['description'] = [
+      '#type' => 'container',
+      'p1' => [
+        '#type' => 'markup',
+        '#markup' => '<p>' . $this->t('Developer synchronization will:') . '</p>',
+      ],
+      'list' => [
+        '#theme' => 'item_list',
+        '#items' => [
+          'Create Drupal users for any Apigee Edge developers that are in this Drupal system',
+          'Create developers in Apigee Edge for all users in this Drupal system that are not already in Apigee Edge',
+        ],
+      ],
+      'p2' => [
+        '#type' => 'markup',
```

Comment:
There is no need to site type if you use markup form element. (This has not changed since D7.)

Reply:
Btw, if you only need a `p` tag use html_tag render element instead.
https://github.com/drupal/core/blob/17557c97a816bc78e70989a86064c562bc27027f/modules/system/tests/modules/render_array_non_html_subscriber_test/src/RenderArrayNonHtmlSubscriberTestController.php#L22

Reply:
Thanks for info, new commit has this change